requirements.txt

```diff
@@ -1,5 +1,7 @@
-aiogram==2.6.1
+python-telegram-bot===12.7
```

Comment:
nip. Use alphabetical order in such files.

Reply:
fixed it